### PR TITLE
All options and more

### PR DESCRIPTION
**Pull request by sharevb**
Corrected almost all opened issues, added almost all options from docker run -h
Read all doc of Docker compose and Docker run

## Checklist

- [x ] Added a test to cover all behaviours
- [x ] Verified some commands and multiline support
- [ x] Behaves according the documentation as far as understood

### CODE REVIEW

File: packages/composerize/__tests__/mappings.test.js [hunk 1]
```diff
@@ -168,3 +169,771 @@ test('--ulimit (https://github.com/magicmark/composerize/pull/262)', () => {
...
+            nginx:
+                healthcheck:
+                    test: healthcheck.sh
+                    interval: 60s
+                    timeout: 10s
+                    start_period: 30s
+                    retries: '2'
+                image: 'nginx:latest'"
+      `);
+});
```
Comment (magicmark) [inline packages/composerize/__tests__/mappings.test.js:939]: hmm github doesn't like the line ending here

try running eslint? https://github.com/magicmark/composerize/blob/ff59318b1d6318d1d88616f6f1f77a5f3904035d/package.json#L13

**Review by magicmark — APPROVED** [review]
🤯 amazing.

thank you so much for working on this! I'll follow up with you via email to discuss more :)

File: packages/composerize/__tests__/mappings.test.js (hunk 1 above)
Comment (sharevb) [reply packages/composerize/__tests__/mappings.test.js:939]: Did it, will retry, may be not the same version

File: packages/composerize/src/mappings.js [hunk 2]
```diff
@@ -38,7 +42,7 @@ export type ArrayComposeEntry = {
 export type KVComposeEntry = {
     path: string,
     value: {
-        [string]: string | number,
+        [string]: string | number | any,
```
[the remark this answers is not in the document]
Comment (sharevb) [reply packages/composerize/src/mappings.js:45]: Yes, you are right, but as far as my knowledge of JS (with flow), it is kind of dynamic

File: packages/composerize-website/src/components/TextInput.js [hunk 3]
```diff
@@ -3,8 +3,9 @@ import styled from 'styled-components';
 
 import CBox from './CBox';
 
-const StyledInput = styled.input`
+const StyledInput = styled.textarea`
     ${CBox}
+	height:8em;
```
Comment (magicmark) [inline packages/composerize-website/src/components/TextInput.js:8]: this introduces a visual change

Screenshot 2023-10-02 at 8 13 48 AM

I think most commands would probably be one liners, but I like the idea of making this a textarea for sure so folks can drag it down

maybe we can start this off with a `rows={1}` or 2 instead so it's not taking up quite as much room by default?

**Review by magicmark — DISMISSED** [review]

File: packages/composerize-website/src/components/TextInput.js (hunk 3 above)
Comment (sharevb) [reply packages/composerize-website/src/components/TextInput.js]: Set 3 lines, may be could also add a second textarea to merge an existing compose with a new docker run command

File: packages/composerize/__tests__/mappings.test.js (hunk 1 above)
Comment (sharevb) [reply packages/composerize/__tests__/mappings.test.js]: eslint made

File: packages/composerize/src/mappings.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (sharevb) [reply packages/composerize/src/mappings.js:45]: may  be will update some flow typing later, currently eslint ok and coverage at 100% so delayed

**Comment by sharevb** [comment]
Merged via #571 
@cdrage  : npm package 1.4.1 published

**Comment by cdrage** [comment]
> Merged via #571 @cdrage : npm package 1.4.1 published

You're the best, thank you.

**Pull request closed without merging**